src/components/Image.js

```diff
@@ -80,7 +80,11 @@ export const Image = ({
         ? CacheManager.get(sourceProp.uri)
             .getPath()
             .then((path) => {
-              mounted && setSource({ uri: path ?? NO_IMAGE.uri });
+              mounted &&
+                setSource({
+                  uri:
+                    __DEV__ && device.platform === 'android' ? sourceProp.uri : path ?? NO_IMAGE.uri
```

Comment:
in case of `__DEV__ && device.platform === 'android'` we can directly return `sourceProp.uri` without going through `CacheManager.get(sourceProp.uri)`, right?!

Reply:
Yes we can. But I didn't want to add an extra if check for this, so I wanted to do it here. But I will update the code to increase the understandability of the code.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/9aeaeab544c1cbede401fa2322a0fcb70c077ebf

Reply:
it is not just about understandability but about the async process (get image from cache) we do not need to run. we can save this effort.